composer.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "openconext/monitor-bundle",
   "type": "symfony-bundle",
-  "description": "A Symfony 3 bundle that facilitates health and info endpoints to a Symfony application. The bundle is backwards compatible with Symfony 2 projects.",
+  "description": "A Symfony 4 bundle that facilitates health and info endpoints to a Symfony application. The bundle is backwards compatible with Symfony 2 projects.",
   "keywords": ["SURFnet", "StepUp", "OpenConext", "monitoring", "health"],
   "license": "Apache-2.0",
   "minimum-stability": "stable",
   "require": {
     "php": ">=5.4,<8.0-dev",
-    "symfony/dependency-injection": ">=2.7,<4",
-    "symfony/framework-bundle": ">=2.7,<4",
```

Comment:
Aside from wanting to drop SF3 support, do these changes really break SF3 support? They seem very 3 compatible. If this is the case, we might get away with a non-BC breaking change here by leaving the SF3 support in place?

Reply:
I removed the support for Symfony prior to 3.4. This isn't currently supported by Symfony either.